### ISSUE TITLE
Add filters to cast variables

### ISSUE THREAD

**Issue by benface** [marionnewlevant/craft-twig_perversion#7]
It would be great to have filters to cast variables as different types, which I suppose is missing from Twig by design:

- `|string` - convert value to String (text)
- `|float` - convert value to decimal number
- `|int` - convert value to a number (no decimal)
- `|bool` - convert value to boolean (true/false)
- `|object` - convert value to object (from array)
- `|array` - convert value to array (from object)

**Comment by lindseydiloreto** [comment]
My two cents... the array filter should just be `array`. No other filters begin with "to".

**Comment by benface** [comment]
Agreed, not sure why I wrote that. Just edited ^

**Comment by lindseydiloreto** [comment]
So believe it or not... I actually wrote a small plugin for this a few months ago. But I didn't release it, because I didn't know if the demand was there (and didn't feel like supporting it). 😄 

I've just submitted a PR, porting in the work I did separately. Everything I did has been included here.

Two important notes...

 - For each filter, I made a one-letter abbreviated version (`string` and `s`).
 - I didn't include the `object` and `array` filters.

Feel free to add those last two filters! And if you feel like the one-letter aliases are overkill, feel free to pull them out. 👍
